View.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime
+
+
+class View:
```

Comment:
What is the purpose of this class?

Reply:
For that we only need functions. Class does not add anything to it and can be avoided.